core/src/test/scala/org/http4s/rho/ApiTest.scala

```diff
@@ -1,22 +1,27 @@
 package org.http4s
 package rho
 
-import bits.MethodAliases._
-import bits.ResponseGeneratorInstances._
-
-import org.http4s.rho.bits.{TypedHeader, PathTree, SuccessResponse, FailureResponse}
-import org.http4s.headers.{`Content-Length`, ETag}
+import org.http4s.headers.{ETag, `Content-Length`}
+import org.http4s.rho.bits.MethodAliases._
 import org.http4s.rho.bits.RequestAST.AndRule
-
+import org.http4s.rho.bits.ResponseGeneratorInstances._
+import org.http4s.rho.bits._
 import org.specs2.mutable._
-import shapeless.HNil
-import scalaz.{-\/, \/-}
+import scodec.bits.ByteVector
+import shapeless.{HList, HNil}
+
 import scalaz.concurrent.Task
 import scalaz.stream.Process
-import scodec.bits.ByteVector
+import scalaz.{-\/, \/-}
 
 // TODO: these tests are a bit of a mess
 class ApiTest extends Specification {
+
+  def runWith[T <: HList, F](exec: RouteExecutable[T])(f: F)(implicit hltf: HListToFunc[T, F]): Request => Task[Response] = {
```

Comment:
The name 'runWith' doesn't read as well in Polish notation as it did in infix.  I suggest either rename it or maybe make it an extension method instead so the syntax stays the same.